spark/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -1667,13 +1667,6 @@ trait DeltaSQLConfBase {
   // Clustered Table
   //////////////////
 
-  val DELTA_CLUSTERING_TABLE_PREVIEW_ENABLED =
```

Comment:
Is it safe to remove this config? Is it only used in testing

Reply:
It's a config that warns user when using clustered table in Delta 3.1. They'll need to explicitly turn on this config to use clustered table. In tests it's ignored.

It's safe to remove since it's only used as guard for preview feature.